pkgs/by-name/cp/cpuinfo/package.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  buildBazelPackage,
```

Comment:
remove this, it seems haven't been use in your file.

Reply:
Thanks - sorry I got so used to linter checking my leftovers :sweat_smile: 

Removed. Thanks!